microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/jdt/internal/core/providers/MicroProfileRegisterRestClientProvider.java

```diff
@@ -131,6 +138,20 @@ protected void processAnnotation(IJavaElement javaElement, IAnnotation registerR
 				super.addItemMetadata(collector, MP_REST_CLASS_REFERENCE_TYPE + "/mp-rest/providers",
 						"java.lang.String", docs, null, null, null, null, null, false);
 
+				// TODO :
+				// com.mycompany.remoteServices.MyServiceClient/mprest/providers/com.mycompany.MyProvider/priority
+				// will override the priority of the provider for this interface.
+
+				// /mp-rest/connectTimeout
+				docs = "Timeout specified in milliseconds to wait to connect to the remote endpoint.";
+				super.addItemMetadata(collector, MP_REST_CLASS_REFERENCE_TYPE + "/mp-rest/connectTimeout",
+						"java.lang.Integer", docs, null, null, null, null, null, false);
+
+				// /mp-rest/readTimeout
+				docs = "Timeout specified in milliseconds to wait for a response from the remote endpoint.";

```

Comment:
there's no validation error if the value is not a number

Reply:
should be fixed.